app/src/components/common/spinner/index.tsx

```diff
@@ -1,8 +1,7 @@
 import React, { HTMLAttributes } from 'react'
 import styled, { keyframes } from 'styled-components'
 
-import SpinnerSVG from './img/spinnerMain.svg'
```

Comment:
If we aren't using the `spinnerMain.svg` or `spinnerMainBig.svg` files anymore we can probably delete them. Thoughts?

Reply:
yeah definitely good catch!